hphp/runtime/vm/jit/vasm-ppc64.cpp

```diff
@@ -1644,6 +1680,138 @@ void fixVptrsForPPC64(Vunit& unit) {
 ///////////////////////////////////////////////////////////////////////////////
 } // anonymous namespace
 
+#include "hphp/util/logger.h"
```

Comment:
@daxtens Why did you include this? After merging to new upstream code this week we had to remove this otherwise it doesn't compile